pkg/common/cliprinter/cliprinter.go

```diff
@@ -0,0 +1,136 @@
+package cliprinter
+
+import (
+	"io"
+	"os"
+
+	"github.com/spiffe/spire/pkg/common/cliprinter/internal/errorjson"
+	"github.com/spiffe/spire/pkg/common/cliprinter/internal/errorpretty"
+	"github.com/spiffe/spire/pkg/common/cliprinter/internal/protojson"
+	"github.com/spiffe/spire/pkg/common/cliprinter/internal/protopretty"
+	"github.com/spiffe/spire/pkg/common/cliprinter/internal/structjson"
+	"github.com/spiffe/spire/pkg/common/cliprinter/internal/structpretty"
+	"google.golang.org/protobuf/proto"
+)
+
+// Printer is an interface for providing a printer implementation to
+// a CLI utility.
+type Printer interface {
+	MustPrintError(error)
+	MustPrintProto(...proto.Message)
+	MustPrintStruct(...interface{})
```

Comment:
Hmm, with the way this is now organized (with the internal stuff), maybe we can elide the `Must` prefix on these names. Whaddya think?

Reply:
I wondered the same ... but then I thought 🤔 it's kind of weird for a non-"Must" function to call os.Exit

I could go either way!